notNeededPackages.json

```diff
@@ -3742,6 +3742,10 @@
             "libraryName": "riot",
             "asOfVersion": "4.1.0"
         },
+        "rn-swipeable-panel": {
+            "libraryName": "rn-swipeable-panel",
+            "asOfVersion": "1.2.2"
```

Comment:
Couldn't find any proof of this but types first started appearing on v1.2.1 (tried by installing the package myself)
```suggestion
            "asOfVersion": "1.2.1"
```

Reply:
Not possible, there is already `@types/rn-swipeable-panel@1.2.1` published on npm. I simply took the next version.

Reply:
This is supposed to be the number of the version that added type declarations for this package, it's not the version of the `@types` package

Already told you this twice on two other PRs, please...

Reply:
The checks won't pass. You can try it out yourself with npm run test-all. This is actually supposed to be the version of the @types package that will be published with the obsoleted warning.

Reply:
Yeah my bad